pykafka/producer.py

```diff
@@ -18,16 +18,17 @@
 limitations under the License.
 """
 __all__ = ["Producer", "AsyncProducer"]
+from collections import defaultdict, deque, namedtuple
 import itertools
 import logging
+import threading
```

Comment:
Don't use `threading`. The handler exists to satisfy all of these objects while allowing us to swap out the implementation when we work on gevent compatibility. (e.g. `handler.Lock`)